dev/com.ibm.ws.security.openidconnect.clients.common/src/com/ibm/ws/security/openidconnect/client/jose4j/util/Jose4jUtil.java

```diff
@@ -429,4 +431,25 @@ ProviderAuthenticationResult checkForReusedJwt(ConvergedClientConfig clientConfi
         return null;
     }
 
+    public static Map<String, Object> getThirdPartyClaims(String thirdPartyToken, Set<String> allowedThirdPartyClaims) {
```

Comment:
I don't think this is ultimately the right class for this method. We can find a more appropriate class and move this method later.